app/models/project.rb

```diff
@@ -1,10 +1,13 @@
 class Project < ActiveRecord::Base
+  before_save :set_github_repo
   belongs_to :organization
   acts_as_ordered_taggable
   acts_as_ordered_taggable_on :technologies, :causes
 
-  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved
-  validates_presence_of :name, :github_repo
+  attr_accessor :git_url
+  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved, :git_url
+  validates_presence_of :name, :git_url
```

Comment:
Also - I know `Project#github_url` is already taken as a method, but it would be great if you'd consider using something more descriptive for this attribute, e.g. `:submitted_github_url` or `:github_repo_url`? (Okay, so, that last one isn't *much* more descriptive...)